allennlp/data/tokenizers/basic.py

```diff
@@ -0,0 +1,33 @@
+from typing import List
+
+from overrides import overrides
+
+from allennlp.common import Params
+from allennlp.data.tokenizers.token import Token
+from allennlp.data.tokenizers.tokenizer import Tokenizer
+
+@Tokenizer.register("basic")
+class BasicTokenizer(Tokenizer):
```

Comment:
BasicTokenizer to me implies just tokenization w/o BOS/EOS tokens -- so can you rename this to something like `TokenizerEOS` or `BasicTokenizerWithEOS` ?

Reply:
Yes, ```BasicTokenizerWithEOS``` sounds good to me